tests/saw/spec/SHA256.cry

```diff
@@ -161,8 +161,8 @@ SHA256Update : {n} (fin n) => SHA256State -> [n][8] -> SHA256State
 SHA256Update sinit bs = ss!0
   where ss = [sinit] # [ SHA256Update1 s b | s <- ss | b <- bs ]
 
-update : {a, b, c} (fin c, c >= width (2 ^^ c - 1)) => [b]a -> [c] -> a -> [min b (2 ^^ c)]a
-update a i x = [ if j == i then x else e | e <- a | j <- [0 ..] ]
+//update : {a, b, c} (fin c, c >= width (2 ^^ (c - 1))) => [b]a -> [c] -> a -> [min b (2 ^^ c)]a
```

Comment:
Should these be uncommented?

Reply:
This function became a primitive in recent versions of Cryptol. Removing that line would probably be better than commenting it. I'll do that.

Thanks for the careful review!